backend/app/livekit_voice_agent.py

```
@@ -0,0 +1,235 @@
+                    "timestamp": None,
+                    "interim": False
+                }
+                logger.info(f"📤 Sending user speech data: {data_payload}")
+                
+                # Send data synchronously
+                ctx.room.local_participant.publish_data(
+                    json.dumps(data_payload).encode('utf-8'),
+                    reliable=True
+                )
+                logger.info(f"📤 Sent user speech to frontend: '{user_text}'")
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)